### ISSUE TITLE
Nv7 Bot commands mostly aren't slash commands

### ISSUE THREAD

**Issue by Nv7-GitHub** [Nv7-GitHub/Nv7Haven#9]
Needs converting to slash command format, also update `discord/cmdutil.go` to be same as `eod/cmdutil.go`

**Comment by rayzchen** [comment]
bit outdated

**Comment by Nv7-GitHub** [comment]
True, but I will convert to slash commands whenever I start working on Nv7 Bot

**Comment by rayzchen** [comment]
whoops i thought this was eode, ignore my comment
